### ISSUE TITLE
Comment typo

### ISSUE THREAD

**Issue by mbinary** [milesial/Pytorch-UNet#105]
https://github.com/milesial/Pytorch-UNet/blob/de7507ff08510b48e6a0e11da849e0d1c94d3ac8/train.py#L147

This line should be `    #   - For 2 classes, use n_classes=2`, right?

**Comment by milesial** [comment]
when n_classes=1, the output is a single layer sigmoid.
when n_classes=2, the output is a 2-layer softmax.
In both cases there is only one degree of freedom per pixel so they are equivalent, but I would recommend just using n_classes=1.
